contracts/sol6/KyberStorage.sol

```diff
@@ -14,7 +15,7 @@ import "./utils/PermissionGroupsNoModifiers.sol";
  *   - Stores feeAccounted data for reserve types
  *   - Record contract changes for network, matchingEngine, feeHandler, reserves, network proxies and kyberDAO
  */
-contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers {
+contract KyberStorage is IKyberStorage, PermissionGroupsNoModifiers, Utils5 {
```

Comment:
Utils5 is used? o.O

Reply:
Will remove it, seems we only use ETH address

Reply:
Back to use Utils5 as its size is quite small